README.md

```diff
@@ -26,13 +26,62 @@ Grails App >= 3.3.x use `v1.0.0` and <=3.2.x use `v0.2.4`
 
 
  * Create a directory `migrations` under `grails-app` and place all your migrations here in JavaScript or XML files.
- 
+
  * For logging, add `grails.mongeez` package to your logger configuration. Example:
 
 ```
 logger('grails.mongeez', DEBUG, ['STDOUT'], false)
 ```
 
+## Configurations for protected database
+
+Mongeez internally calls db.eval() method to execute queries.
+
+“If authorization is enabled, you must have access to
+all actions on all resources in order to run eval. Providing such access is not recommended, but if your organization
+requires a user to run eval, create a role that grants anyAction on anyResource.
+Do not assign this role to any other user.”
+https://docs.mongodb.com/manual/reference/method/db.eval/#access-control
```

Comment:
It would be better to use blockquote when you are quoting something.
```
> If authorization is enabled, you must have access to all actions on all resources in order to run eval. Providing such access is not recommended, but if your organization requires a user to run eval, create a role that grants anyAction on anyResource. Do not assign this role to any other user.

Reference: https://docs.mongodb.com/manual/reference/method/db.eval/#access-control
```

Reply:
Done.